### ISSUE TITLE
Image Map not working for me (duc cgi)

### ISSUE THREAD

**Issue by jotham** [zevv/duc#87]
I'm not sure if I've misconfigured duc cgi or messed something else up.

Google Chrome and Firefox (I am on Windows 7) are both producing a URL in the form of http://localhost:8080/cgi/duc.cgi?cmd=index&path=/home/jotham&?725,317 which looks a little incorrect.

Putting in some print debugging into do_index in cmd-cgi.c
```
printf("<!-- x %d, y %d -->\n", x, y);
```

I'm getting the expected output :
```
<!-- x 725, y 317 -->
```

But the CGI isn't going any deeper into the file system.

What have I set up wrong?  

Apache configuration
```
<VirtualHost 127.0.0.1:8080>
   ScriptAlias /cgi/ /home/jotham/cgi-bin/
   <Directory /home/jotham/cgi-bin>
      Options +ExecCGI
      AddHandler cgi-script .cgi
   </Directory>
   ErrorLog ${APACHE_LOG_DIR}/cgi-bin.error.log
   LogLevel warn
   CustomLog ${APACHE_LOG_DIR}/cgi-bin.log combined
</VirtualHost>
```

/home/jotham/.duc.db
```
-rw-rw-r--   1 jotham www-data  2.3M May 11 20:07 .duc.db
```

/home/jotham/cgi-bin/duc.cgi 
```
#!/bin/sh
/usr/local/bin/duc cgi -d /home/jotham/.duc.db
```

Kind regards,

**Comment by zevv** [comment]
* On 2015-05-11 10:58:25 +0200, Jotham Read wrote:
 
> I'm not sure if I've misconfigured duc cgi or messed something else up.

No, a stupid mistake from my side: I accidentally checked in unrelated
test code with a commit, which broke CGI coordinate lookups, combined
with lousy testing. 

Fixed in git master, and the new 1.2 release I just put out.

-- 
:wq
^X^Cy^K^X^C^C^C^C

**Comment by jotham** [comment]
Works great, cheers man!
